common/pluginData.go

```diff
@@ -17,9 +17,9 @@ package common
 import "github.com/apid/apid-core"
 
 var PluginData = apid.PluginData{
-	Name:    "apidVerifyAPIKey",
-	Version: "0.1.0",
+	Name:    "apidApiMetadata",
```

Comment:
Would it be beneficial to use global constants (or is it too expensive to have global constants)?

Reply:
Yes, we can have constants.